examples/deadline/EC2-Image-Builder/README.md

```diff
@@ -0,0 +1,60 @@
+# RFDK Sample Application - EC2 Image Builder
+
+Keeping software updated on custom AMIs can be a pain if you're creating them manually. The EC2 Image Builder service has matured to the point that it is a viable option for automating the process. While the support in CloudFormation and CDK is still a bit lacking, it is usable and can be worked into an RFDK app to build the required AMIs for the worker fleet on the fly.
+
+This is a sample RFDK application that utilizes [EC2 Image Builder](https://docs.aws.amazon.com/imagebuilder/latest/userguide/what-is-image-builder.html) to install and configure Deadline onto an AMI that can then be used as the image for a worker fleet. The process of creating an AMI can be lengthy, sometimes extending the deployment of a render farm by 30 minutes, but if you have multiple custom AMI's that you maintain, you may find it easier to let your RFDK application install new versions of Deadline onto them, rather than having to do it manually.
+
+Another option is using EC2 Image Builder from the AWS Console to create an image pipeline. We don't touch on that here, but you may want to see the [Create an image pipeline user guide](https://docs.aws.amazon.com/imagebuilder/latest/userguide/start-build-image-pipeline.html) and see if this option would be a better fit for you.
+
+---
+
+_**Note:** This application is an illustrative example to showcase some of the capabilities of the RFDK. **It is not intended to be used for production render farms**, which should be built with more consideration of the security and operational needs of the system._
+
+---
+
+## Architecture
+
+This sample application uses an [EC2 ImageBuilder Image](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-image.html) to create an AMI in the DeadlineImage stack that is used by the WorkerInstanceFleet in the same stack. One important part of the design of this app to note is that we are making sure the farm is running the same version of Deadline everywhere by using the version of Deadline from the Deadline container recipes retrieved by the `stage-deadline` script (refer to the [All-In-AWS-Infrastructure-Basic](../All-In-AWS-Infrastructure-Basic/README.md) for more details) in all of the constructs that need to install Deadline.
```

Comment:
I forgot to update this when I switched the example over from using the Deadline recipes to using the images, I'll have to remove the reference to `stage-deadline` in this paragraph and in `package.json`.